docs/concepts/typings.md

```diff
@@ -4,4 +4,103 @@ title: Type hints
 
 # Type hints
 
-...
+Type hints are a modern feature of Python available since 3.5 whose existence is heavily influenced by the features of type-safe languages such as Rust. To learn more about type hints in Python, see [Real Python’s Type hinting walkthrough](https://realpython.com/lessons/type-hinting/).
```

Comment:
Can we add explicit line wrapping (probably ~80 or ~120 chars) to this file.

Reply:
Will go with 120 for now subject to any further requests